src/main/java/mchorse/mclib/ClientProxy.java

```diff
@@ -22,13 +23,17 @@
 @SideOnly(Side.CLIENT)
 public class ClientProxy extends CommonProxy
 {
+    public static KeybindConfig keybinds;
```

Comment:
At the moment I'm ~~busy~~ lazy to take a deep dive into this, but if you could summarize what this commit is about? 👀

Reply:
This commit adds a new config page that allows you to configure hotkeys for each interface in McLib and its submods.
I tried to have each Keybind object add itself to ClientProxy.keybinds at construct time, but, I have to say this really isn't the most appropriate way to do it.

Reply:
> This commit adds a new config page that allows you to configure hotkeys for each interface in McLib and its submods.
> I tried to have each Keybind object add itself to ClientProxy.keybinds at construct time, but, I have to say this really isn't the most appropriate way to do it.

Wow! That’s really cool! 😮